tests/test_nodes.py

```diff
@@ -103,6 +103,7 @@ def test_basic(self):
         assert(
             sorted(self.test_obj.property_names) ==
             [
+                Cell.HOLDING_CURRENT,
```

Comment:
no THRESHOLD_CURRENT here ?

Reply:
These are optional attributes; I have put only `holding_current` attribute to `dynamics_params` in the test data.

Reply:
ok make sense.